### PR TITLE
Shooter inclinator tuning

### PR DESCRIPTION
**Pull request by mlists**
I would like to do a bit more testing before we merge this but it runs reasonably on the test rig.

### CODE REVIEW

File: components/shooter.py [hunk 1]
```diff
@@ -60,13 +59,20 @@ def shoot(self) -> None:
...
+    @feedback
+    def encoder_raw(self):
+        return self.inclinator_encoder.getOutput()
+
     @feedback
     def at_inclination(self) -> bool:
         return self.inclinator_controller.atGoal()
 
     @feedback
     def inclination_angle(self) -> float:
```
Comment (auscompgeek) [inline components/shooter.py:71]: nit: we should document what coordinate system we're using.

**Comment by mlists** [comment]
This is ready for review now

**Review by SebZanardo — COMMENTED** [review]
LGTM

**Review by SebZanardo — APPROVED** [review]
LGTM

**Pull request merged**